ansible/roles/prometheus-server/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+# Prometheus
```

Comment:
The comment here can be removed as the variable names that follow are descriptive enough.

Reply:
Comment removed.